opendevin/core/config.py

```diff
@@ -82,7 +82,6 @@ class AppConfig(metaclass=Singleton):
     )
     run_as_devin: bool = True
     max_iterations: int = 100
-    remind_iterations: bool = False
```

Comment:
If we remove this, do we want to assume this will always be true for CodeAct?

I have a mixed feeling about always enabling it. I am not sure if it is the best route for open-ended tasks people typically do (e.g., people may send instructions/requests for new tasks along the way - and maybe there's only 20 steps budget left, and the LLM has super stressed).

But let's get it in for the time being - We didn't really have the memory/context condenser for the agent to work on really long-horizon tasks, so setting a budget is reasonable. Let's worry about it after then.